### PR TITLE
Dshop control panel unity

### PR DESCRIPTION
**Pull request by mikeshultz**
### Description:

Unification of control panels and some other stuff.

- Moved around ports because various conflicts
- Backend auth codes should be able to be put into config.json instead of just at dapp built-time
- Require login if shop being edited is on backend
- conditionally create shop on backend
- Config URL linked in `<link>` tag in html head
- Config is parsed using this tag
- fix URL mashing
- Some error handling

Not super happy about this UX, but it's a step.

### Checklist:

- [ ] Test your work and double-check to confirm that you didn't break anything
- [ ] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [ ] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

### CODE REVIEW

File: infra/dshop-control-client/src/components/Edit/DeployWizard/Password.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from 'react'
 import { useStoreState } from 'pullstate'
-import axios from 'axios'
+import axios from 'utils/axiosWithCredentials'
```
Comment (tomlinton) [inline infra/dshop-control-client/src/components/Edit/DeployWizard/Password.js]: There was a reason I separated these, but can't remember exactly why. Does the flow still work if they've never registered?

File: infra/dshop-control-server/src/index.js [hunk 2]
```diff
@@ -60,10 +57,11 @@ app.get('/ingest/:url', async (req, res) => {
 
     return res.json({
       collections,
-      products
+      products,
+      datadir
     })
   } else {
-    console.log('Fetching dShop data for', `${url}`)
+    console.log('Fetching dShop data for')
```
Comment (tomlinton) [inline infra/dshop-control-server/src/index.js]: Remove `for`?

File: infra/dshop-control-client/src/pages/Edit.js [hunk 3]
```diff
@@ -18,19 +18,32 @@ import Products from 'pages/Edit/Products'
 import ProductAdd from 'pages/Edit/ProductAdd'
 import ProductEdit from 'pages/Edit/ProductEdit'
 import Settings from 'pages/Edit/Settings'
-import Navigation from 'components/Edit/Navigation'
+import SignIn from 'pages/SignIn'
+//import Navigation from 'components/Edit/Navigation'
+import Navigation from 'components/Navigation'
```
Comment (tomlinton) [inline infra/dshop-control-client/src/pages/Edit.js]: Doesn't look like the component was moved?

File: infra/dshop-control-client/src/pages/App.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ import Manage from './Manage'
 import Landing from './Landing'
 import ProcessShop from './ProcessShop'
 import SignIn from './SignIn'
+import SignOut from './SignOut'
```
Comment (tomlinton) [inline infra/dshop-control-client/src/pages/App.js]: Doesn't exist?

File: infra/dshop-control-client/src/components/Edit/DeployWizard/Password.js (hunk 1 above)
Comment (mikeshultz) [reply infra/dshop-control-client/src/components/Edit/DeployWizard/Password.js]: It was actively breaking things without that setting.  And it shouldn't cause any issues.  I believe it just means cookies and certain headers can be used, which you need when you're logging in to things or depending on cookies for auth.

File: infra/dshop-control-client/src/pages/Edit.js (hunk 3 above)
Comment (mikeshultz) [reply infra/dshop-control-client/src/pages/Edit.js]: Yikes, forgot to commit a few files.  Thanks for catching this.  Need to clean up my working tree.

File: infra/dshop-control-client/src/pages/App.js (hunk 4 above)
Comment (mikeshultz) [reply infra/dshop-control-client/src/pages/App.js]: Also forgot to commit

**Pull request merged**